content/en/security_platform/application_security/_index.md

```diff
@@ -1,17 +1,52 @@
 ---
-title: Monitoring Threats with Datadog Application Security
+title: Application Security
 kind: documentation
+description: Monitor threats targeting production system, leveraging the execution context provided by distributed traces.
+is_beta: true
+disable_sidebar: true
 further_reading:
 - link: "/security_platform/guide/how-appsec-works/"
   tag: "Guide"
-  text: "How Datadog Application Security Monitoring Works"
+  text: "Learn how Application Security works"
```

Comment:
I don't think this will be there for the release. Maybe instead, do we know what the Release Notes link will be? I think we're supposed to add that to landing pages now.

Reply:
Good question, @arnaudbreton do we have a link handy for the release notes? If not, we can add after publishing to direct folks to more details.